mtp_api/apps/mtp_auth/models.py

```diff
@@ -88,6 +88,13 @@ def assign_to_user(self, user):
             user.groups.add(group)
 
 
+class JobInformation(models.Model):
```

Comment:
one thing i'd suggest, but it's no big deal, is subclassing TimeStampedModel. that way, you know the date/time when someone entered their job title. i think that'd be helpful. if you do this, you can then add 'modified' to the admin `list_display` and add an `ordering`.

Reply:
Great - I think the ordering is most useful so that's been added.